src/index.js

```diff
@@ -5,7 +5,7 @@ import Dynamic from './modules/dynamic.js';
 import setLocalStorage from './modules/localStorage.js';
 import refresh from './modules/refresh.js';
 import render from './modules/render.js';
-import { scoresInputBtn, scoresRefresh } from './modules/variables.js';
+import { scoresInputBtn, scoresRefresh, soundtrack } from './modules/variables.js';
 import './style.css';
 
 scoresInputBtn.addEventListener('click', async () => {
```

Comment:
[OPTIONAL]

- I would recommend clearing the input fields after submitting the button so that users can understand that their data has been submitted.

Reply:
Thanks for the tip! I've made it already! 😄 ✔️